### ISSUE TITLE
Very low infill, will not update when changed

### ISSUE THREAD

**Issue by Rohrtech** [supermerill/SuperSlicer#4284]
### What happened?

Infill % is extremely low. Does not change when infill % is modified.
![image](https://github.com/supermerill/SuperSlicer/assets/89417131/fcf7868b-aa52-41b2-b01f-d2a9b1b445d1)
![image](https://github.com/supermerill/SuperSlicer/assets/89417131/7c1a3e78-8a95-4679-9e47-f1f127b48012)

### Project file & How to reproduce

Slice file with Ubuntu 2.5.59 appimage on Ubuntu 20.04.6 LTS
[Project1.zip](https://github.com/supermerill/SuperSlicer/files/15448256/Project1.zip)

### Version

2.5.59

### Operating system

Ubuntu 20.04.6 LTS

### Printer model

CR-10S

**Comment by supermerill** [comment]
probably duplicate of #4243

**Comment by Rohrtech** [comment]
Yep, seems to be the case. Sorry for the duplicate! Will wait to upgrade until the next release.
